daphne/src/lib.rs

```diff
@@ -225,6 +236,45 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_window_size`, `min_duration_start`, and `max_duration_end`
+//            should be reflected to the DAP spec.
+#[derive(Deserialize, Serialize)]
+#[serde(try_from = "ShadowDapGlobConfig")]
+pub struct DapGlobConfig {
+    /// Maximum window size permitted in CollectReq.
+    /// Prevents Collectors from requesting wide range or reports.
+    pub max_window_size: u64,
+
+    /// Lower bound of an acceptable batch window in CollectReq.
+    /// Batch intervals must start after `min_duration_start`.
+    pub min_duration_start: u64,
+
+    /// Higher bound of an acceptable batch window in CollectReq.
+    /// Batch intervals must end before `max_duration_end`.
+    pub max_duration_end: u64,
```

Comment:
Is this a duration (i.e., a number of seconds) or a timestamp (i.e., number of seconds after UNIX epoch)?

Reply:
This is a number of seconds.